test/testfixture/testfixture.go

```diff
@@ -218,6 +221,7 @@ func newFixture(db *gorm.DB, isolatedCreation bool, recipeFuncs ...RecipeFunctio
 		makeWorkItems,
 		makeComments,
 		makeWorkItemLinks,
+		makeTrackers,
```

Comment:
Move close to top in the section where identities and work item link cats are created.

Reply:
done as a part of this commit -> a585aef